core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -154,7 +154,7 @@ oppia.controller('ExplorationEditor', [
           successCallback();
         }
 
-        stateEditorTutorialFirstTimeService.init(
+        $scope.firstTime = stateEditorTutorialFirstTimeService.init(
```

Comment:
Why do you need to store the return value? I don't think it's used, but I may have missed something.